intermediate_source/inductor_debug_cpu.py

```diff
@@ -0,0 +1,622 @@
+# -*- coding: utf-8 -*-
+
+"""
+Inductor CPU backend debugging and profiling
+============================================
+
+**Author**: `Liao Xuan <https://github.com/Valentine233>`_, `Zhu Haozhe <https://github.com/zhuhaozhe>`_
+"""
+
+#########################################################################
+# Overview
+# --------
+#
+# This document is intended to introduce the usage, debugging and performance profiling for ``torch.compile`` with Inductor CPU backend.
+#     1. For the usage, we will show how to print debugging loggings and how to inductor an in-depth analysis with config parameters.
```

Comment:
`how to inductor an in-depth analysis with config parameters`?

It's confusing